src/components/ContentPost/ContentPost.tsx

```diff
@@ -20,32 +14,26 @@ const ContentPost = ({
       href={link}
       className="group flex flex-col overflow-hidden rounded-md w-full bg-grey-100 hover:bg-grey-200 border border-grey-200 hover:border-grey-300 cursor-pointer transition duration-300 shadow-md h-full"
     >
-      <div className="relative group">
-        <Image
-          width={300}
-          height={300}
-          alt=""
-          src={`https:${thumb?.fields?.file?.url || ""}`}
-          className="w-full aspect-7/4 block object-cover object-top transition-transform group-hover:scale-102 duration-300"
-        />
-        {labeled && (
-          <div className="absolute top-4 left-4 rounded-lg px-2 py-0.5 bg-gray-100">
-            <p className="font-semibold text-xs">
-              {
-                POST_TYPES[
-                  type as "additional-content" | "data-panel" | "newsletter"
-                ]
-              }
-            </p>
-          </div>
-        )}
+      <Image
+        width={300}
+        height={300}
+        alt=""
+        src={`https:${thumb?.fields?.file?.url || ""}`}
+        className="w-full aspect-7/4 block object-cover object-top transition-transform group-hover:scale-102 duration-300"
+      />
```

Comment:
Would be better to wrap this component into so it doesn't overflow and hides the other components


```suggestion
<div className="w-full overflow-hidden">
        <Image
          width={300}
          height={300}
          alt=""
          src={`https:${thumb?.fields?.file?.url || ""}`}
          className="w-full aspect-7/4 block object-cover object-top transition-transform group-hover:scale-102 duration-300"
        />
      </div>
```

Reply:
This will cause the component to have some styling issues at first, but if you add this on the rest of the components it will solve the issue:

```js
      <div className="flex flex-col grow-1 justify-between">
        <div className="flex flex-row justify-between items-center bg-gray-200 px-5 py-1">
          <p className="font-semibold text-xs">{POST_TYPES[type]}</p>
          <p className="text-grey-600 text-xs">{formattedDate}</p>
        </div>

        <div className="py-4 px-5 flex gap-3 justify-between h-full items-center box-border">
          <p className="line-clamp-2 text-sm font-medium text-wrap">{title}</p>
          <Icon
            className="md:flex rotate-270 size-2 min-w-2"
            id="expand"
            size={9}
          />
        </div>
      </div>

```